ProjectSettings/TagManager.asset

```diff
@@ -7,11 +7,12 @@ TagManager:
   - Item
   - UI
   - Enemy
+  - Robot
   layers:
   - Default
   - TransparentFX
   - Ignore Raycast
-  - 
+  - Floor
```

Comment:
I'm not sure about that. What if we make a smooth surface (eg. sphere) that can act like wall and floor (and ceiling) at the same time?

Reply:
Deleted the layer as it was not used at all.